src/common/pack.h

```diff
@@ -253,6 +253,7 @@
 
 #define PKT_GUIDE		196	/* Server issues Guide-invocation on client-side */
 
+#define PKT_INDICATORS 197
```

Comment:
I know, there are many misaligned constants, but try to align as the one before.

Reply:
fixed